react/src/App.tsx

```diff
@@ -3,13 +3,40 @@ import Header from "./components/Header";
 import ModeSwitcher from "./components/ModeSwitcher";
 
 function App() {
-  const [ mode, setMode] = useState<'light' | 'dark'>('light')
+  const [mode, setMode] = useState<"light" | "dark">("light");
   return (
     <div className={`App sdds-mode-${mode}`}>
-      <Header/>
-      <ModeSwitcher  mode={mode} setMode={setMode} />
+      <Header />
+      <ModeSwitcher mode={mode} setMode={setMode} />
       <main>
         {/* COMPONENTS GO HERE */}
+        <div>
+          <div className="sdds-headline-02">Accordion</div>
```

Comment:
```suggestion
          <div className="sdds-headline-02 sdds-u-pb1">Accordion</div>
```

Just to get some spacing...